app-laptop/thinkfan/thinkfan-0.9.3.ebuild

```diff
@@ -18,17 +18,21 @@ IUSE="atasmart"
 DEPEND="atasmart? ( dev-libs/libatasmart )"
 RDEPEND="${DEPEND}"
 
-S="${WORKDIR}"
-
 src_prepare() {
+	default
+
+	sed -e "s:#!/sbin/runscript:#!/sbin/openrc-run:" \
+		-i rcscripts/thinkfan.gentoo
```

Comment:
Looks upstream already fixed this in [this commit](https://sourceforge.net/p/thinkfan/code/ci/5308ef48209774dd979a92060010c721248606ed/) but It's not yet released.  Latest upstream release is 1.0-beta2 but it lacks openrc service file.